tasks/install-package.yml

```diff
@@ -75,27 +64,26 @@
   shell: "subscription-manager repos --enable=rhel-server-rhscl-7-rpms"
   when: ansible_distribution == "Red Hat Enterprise Linux" and ansible_distribution_major_version == '7'
 
-- name: install latest ondemand
-  package:
-    name: ondemand
-    state: "{{ ondemand_package }}"
+- name: Update cache (Debian)
+  ansible.builtin.apt:
     update_cache: true
-  when: ondemand_package == "latest"
+  when: ansible_os_family == "Debian"
 
-- name: install specific version of ondemand (RHEL)
+- name: Install Open Ondemand (RHEL)
   package:
     name:  "{{ ondemand_package }}"
     state: present
     update_cache: true
     exclude: "{{ ondemand_package_excludes | default([]) }}"
-  when: ondemand_package != "latest" and ansible_os_family == "RedHat"
+  when: ansible_os_family == "RedHat"
 
-- name: install specific version of ondemand (Debian)
+- name: Install Open Ondemand (Debian)
   package:
     name: "{{ ondemand_package }}"
     state: present
     update_cache: true
-  when: ondemand_package != "latest" and ansible_os_family == "Debian"
+    dpkg_options: 'force-confold,force-confdef,force-confnew'
```

Comment:
I'm removing the `latest` task and replacing it with these 2 RHEL and Debian tasks so that I can pass `dpkg_options` for Debian and `exclude` for RHEL no matter what version is being installed.